src/sentry/static/sentry/app/views/organizationIntegrationConfig.jsx

```diff
@@ -0,0 +1,305 @@
+import {Flex, Box} from 'grid-emotion';
+import React from 'react';
+import styled from 'react-emotion';
+import _ from 'lodash';
+
+import {sortArray} from '../utils';
+import {t} from '../locale';
+import AlertLink from '../components/alertLink';
+import AsyncView from './asyncView';
+import Button from '../components/buttons/button';
+import Confirm from '../components/confirm';
+import EmptyMessage from './settings/components/emptyMessage';
+import IndicatorStore from '../stores/indicatorStore';
+import LoadingError from '../components/loadingError';
+import Panel from './settings/components/panel';
+import PanelBody from './settings/components/panelBody';
+import PanelHeader from './settings/components/panelHeader';
+import PanelItem from './settings/components/panelItem';
+import PluginIcon from '../plugins/components/pluginIcon';
+import SettingsPageHeader from './settings/components/settingsPageHeader';
+import marked from '../utils/marked';
+
+const IntegrationIcon = styled.img`
+  height: 32px;
+  width: 32px;
+  border-radius: 3px;
+  display: block;
+`;
+
+const IntegrationName = styled.div`
+  font-size: 16px;
+  margin-bottom: 3px;
+`;
+
+const DomainName = styled.div`
+  color: ${p => p.theme.gray3};
+  font-size: 14px;
+`;
+
+const Details = styled(Flex)`
+  font-size: 15px;
+  line-height: 2.1rem;
+`;
+
+const Description = styled.div`
+  li {
+    margin-bottom: 6px;
+  }
+`;
+
+const AuthorName = styled.div`
+  color: ${p => p.theme.gray2};
+`;
+
+const MetadataLink = styled.a`
+  display: block;
+  margin-bottom: 6px;
+`;
+
+const alertLinkMarkedRenderer = new marked.Renderer();
+alertLinkMarkedRenderer.paragraph = s => s;
+
+function computeCenteredWindow(width, height) {
+  const screenLeft = window.screenLeft != undefined ? window.screenLeft : screen.left;
```

Comment:
if this is copy pasted from somewhere, could you include a link in case someone is trying to debug it in the future?

Reply:
Copied from dcramer:

https://github.com/getsentry/sentry/blob/b0fa526b28df4a4b557c874298e9a2aa97ee3098/src/sentry/static/sentry/app/views/organizationIntegrations.jsx#L81-L97

(this whole other integration component will go away soon)